src/components/sidebar/sidebar-component.tsx

```diff
@@ -284,6 +284,16 @@ class SidebarComponent extends Component<Props, State> {
             })(),
           ],
         });
+      case DiagramType.CommunicationDiagram:
```

Comment:
Should we allow different elements than `ObjectName`s? Some examples contained `Actor`s as well.

Reply:
we don't need actors for now. Objects are enough. Can we display attributes and methods in the objects? (this should work similar to Object Diagrams)